ofrak_patch_maker/ofrak_patch_maker/toolchain/utils.py

```diff
@@ -5,15 +5,19 @@
 from multiprocessing import Pool, cpu_count
 from typing import Optional, Dict, Mapping, Tuple
 
-import magic
-
 from ofrak_patch_maker.toolchain.model import BinFileType, Segment
 from ofrak_type.error import NotFoundError
 from ofrak_type.memory_permissions import MemoryPermissions
 
+_magic = None
+
```

Comment:
In favor of deleting this in conjunction with https://github.com/redballoonsecurity/ofrak/pull/299/files#r1194275565.:

```suggestion
```